README.md

```diff
@@ -66,7 +66,9 @@ ROM CRC32: 1394F57E
 
 A link to the BPS can be found on the [releases page](https://github.com/kirjavascript/TetrisGYM/releases).
 
-The BPS produces a file with an MMC1 header, but it also works when treated as CNROM.
+The BPS produces a file with its header specifying MMC1 with fixed PRG (mapper 1:5), but it also works when specified as CNROM with bus conflicts (mapper 3:2).
+
+If you are using a PowerPak, you will need to install an [alternate N.MAP loader](https://forums.nesdev.org/viewtopic.php?p=283943#p283943) so its header check passes.
```

Comment:
This seems like a regression and makes me wonder if making NES2.0 the default is actually a good idea or not.

Reply:
Official development for the PowerPak ended in 2010 (the thing uses Compact Flash cards, it's that old). I think it's reasonable to assume that owners would be aware of (and have already implemented) compatibility workarounds not limited to NES2.0 headers by this point. Otherwise, the problematic byte can just be reverted to 0 in this case.

Reply:
I'm looking at it this way;

There's no real disadvantage from using an iNES header.
If we switch to NES2.0 we have to add this info to the 'Getting Started' section, which doesn't really apply to any users, and the ones it does apply to are given this extra step.

Another alternative is providing multiple builds with each release. iNES, NES2, Keyboard, etc

Reply:
In that case, we can just leave the problematic byte as $00.